docusaurus.config.js

```diff
@@ -217,7 +217,7 @@ module.exports = {
           showLastUpdateAuthor: true,
           showLastUpdateTime: true,
           editUrl:
-            'https://github.com/testing-library/testing-library-docs/blob/master/docs/',
+            'https://github.com/testing-library/testing-library-docs/blob/master/',
```

Comment:
This does fix the issue, but while we're at it we should probably link directly to the edit page so users don't have to press the edit button.

```suggestion
            'https://github.com/testing-library/testing-library-docs/edit/master/',
```

Reply:
You're right, didn't think about it :)
Thanks!